dds_web/security/auth.py

```diff
@@ -149,19 +151,32 @@ def verify_invite_key(token):
     if not invite:
         raise InviteError(message="Invite could not be found!")
 
-    # Verify temporary key from token claims
-    temporary_key = bytes.fromhex(claims.get("sen_con"))
-    if verify_invite_temporary_key(invite=invite, temporary_key=temporary_key):
-        return temporary_key
+    try:
+        return invite, bytes.fromhex(claims.get("sen_con"))
+    except ValueError:
+        raise ValueError("Temporary key is expected be in hexadecimal digits for a byte string.")
```

Comment:
Is this to avoid assigning the sensitive content to a variable and cleaning up?

Reply:
I mean the reason for not getting the sensitive content in the try-except and then returning if ok

Reply:
sensitive content can be empty, that is handled afterwards. As you can see from ValueError and the raise message, this handles if a non-hex value comes. If we don't attend to it, there will be a useless exception trace saying it couldn't parse bytes from hex etc.